brevo/README.md

```diff
@@ -0,0 +1,62 @@
+## Overview
+
+[Brevo][1] is a cloud-based digital marketing platform designed for creating, sending, and tracking email campaigns, transactional emails, and more. It offers tools for automation and analytics, helping businesses optimize their email marketing strategies and monitor performance.
+
+Integrate Brevo with Datadog to gain insights into Brevo marketing campaign emails and track Brevo performance based on events and other transactional events using [webhooks][2].
+
+## Setup
+
+Follow the instructions below to configure this integration for Brevo Marketing and Transactional events through a Webhook.
+
+### Configuration
+
+#### Webhook configuration for marketing events
+Configure the Datadog endpoint to forward Brevo marketing events as logs to Datadog. For more details, see the Brevo [Marketing webhooks][3] documentation.
```

Comment:
It might be helpful to link to our docs on intake URL's, specifying to select the one appropriate for their DC: https://docs.datadoghq.com/logs/log_collection/?tab=host

Reply:
The customer still has to specific where to send their logs which is based on their Datacenter. We're pointing them to the right URL to point to in this doc:

https://developers.brevo.com/docs/how-to-use-webhooks

Reply:
@apiazza-dd Can you please elaborate, what is the ask here ?

Reply:
I'm looking to add something like this to the instructions: 

To find the correct "URL to Post To" for setting up your webhook in Brevo, please refer to [these docs](https://docs.datadoghq.com/logs/log_collection/?tab=host). For more information on setting up webhooks in Brevo, see the [Brevo webhook documentation](https://developers.brevo.com/docs/how-to-use-webhooks).

Reply:
Sorry, I was confused here, I thought this was push, this looks good, approved!